blog/content/post/2020-04-16-remote_work_post.md

```diff
@@ -0,0 +1,68 @@
+
+Since the recent COVID-19 outbreak we all have been forced to pack our tools
+along with our potential and switch to home office. One may say that for IT it
+is not a milestone; after all our work didn’t change significantly. We can still
+develop solutions of an invariably good quality providing that our remote
+environment has been fully set. Apart from tools there are more factors,
+combined in workflow, of which we are deprived while working remotely for a long
+term. Among endless interruptions and no watchful eyes around, with difficulties
+to connect people directly, there is the question that rises inside the head:
+<b> Ping or not to ping? </b>
+
+
+
+![remote_work01](/img/remote_work01.jpg)
+
+Our team was aware of the difficulties made by the current situation. We were
+prepared for the necessity of a sudden switch from office to remote work,
+treating current situation as a challenge to deepen competences that will allow
+us return to the office in a better form. But still, we do agree that this is
+not an easy situation to cope with. 3mdeb team, asked for the greatest
+difficulties, pointed problem with maintaining work-life balance. We are, after
+all surrounded by the household members, with no possibility to change the
+environment, even in the form of a short walk. We complain about the problem of
+maintaining full concentration among domestic interruptions and feel severely
+the lack of direct contact with co-workers; use of messengers extends sometimes
+the performance of certain tasks. Fortunately, different situation goes with
+motivation among the team. Vast majority estimates their motivation at a high
+level. Strong sense of responsibility makes us effectively solve encountered
+difficulties.
+
+
+##### How do we enhance our effectiveness working remotely
+
+We all agree that effectiveness is based on a good planning and detailed goals.
+Checklists, detailed notes, to-do lists and smart breaks help us to maintain our
+workflow. Some of us effectively use the pomodoro technique, others break the
+eight-hour work system by dividing it into smaller parts. During breaks, we try
+to leave the computer, spending time with the household members, going out into
+the fresh air or doing sports. Responding to encountered difficulties, we adapt
+to the current situation, maintaining the good quality of our work. All team
+agrees that it will definitely be a positive impact. The current situation
+teaches us better work organization, self-discipline and strengthens the sense
+of responsibility. It will be undeniably easier for us to stay focused in the
+work environment with the colleagues sitting nearby. After such a break, it will
+be much easier for us to appreciate the office everyday. What we will miss about
+remote working? Surely the flexibility of working hours and the lack of
+commuting. Nevertheless, we are all waiting for the end of the pandemic and
+quick return to our desks, screens and mugs.
```

Comment:
@KBanecka I like the social feeling and emotion of the post, but we are open-source software and hardware company as well as embedded systems consulting. I would appreciate if you could underline how our beliefs, experience, past project and open technologies that we using every day helping us in remote work productivity.

For example, this is a very good post from well-respected company: https://puri.sm/posts/our-essential-list-of-free-software-for-remote-work/

Reply:
@pietrushnic I'm including your suggestions

Reply:
[commit](https://github.com/3mdeb/news-and-ideas/commit/dd554d2ecb716dae3e9e9afe9585bc912cd927c1#diff-a68390b5dcec9ebb7b7e5492bfbe6cdc)